src/style.css

```diff
@@ -0,0 +1,28 @@
+body {
+  margin: 0;
+}
+
+.stars {
+  display: flex;
+  margin-bottom: 4px;
+}
+
+.stars__star {
+  width: 16px;
+  height: 16px;
+  margin-right: 4px;
+  background-image: url(./images/star.svg);
+  background-repeat: no-repeat;
+}
+
+.stars__star:last-child {
+  margin-right: 0;
+}
+
+.stars--1 .stars__star:first-child,
+.stars--2 .stars__star:nth-child(-n+2),
+.stars--3 .stars__star:nth-child(-n+3),
+.stars--4 .stars__star:nth-child(-n+4),
+.stars--5 .stars__star:nth-child(-n+5) {
+  background: url(./images/star-active.svg);
```

Comment:
you need to overwrite just background-image, but when you use 
'background' rule, you also remove 'background-repeat: no-repeat', so at the end you have such mistake

![image](https://user-images.githubusercontent.com/57598825/130784515-7e74aa77-ca1c-4168-a66f-59a4234eb767.png)


```suggestion
  background-image: url(./images/star-active.svg);
```